modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -5682,6 +5682,7 @@ definition-info=Definition Info
 definition-name=Definition Name
 definition-name-help=Definition name
 definition-of-terms=Definition of Terms
+definition-of-terms-translatable-fields-info=This object includes translatable fields. Notification Terms always use a field's default language.
```

Comment:
just a heads-up but usually when we add new language keys to the code we create a separate commit to add the key and another commit called `buildLang` to generate the key in other languages. you don't need to change on this PR, but is a pattern we tend to follow here!